scanner/src/funTest/kotlin/provenance/AbstractPackageProvenanceStorageFunTest.kt

```diff
@@ -21,6 +21,8 @@ package org.ossreviewtoolkit.scanner.provenance
 
 import io.kotest.core.listeners.TestListener
 import io.kotest.core.spec.style.WordSpec
+import io.kotest.matchers.collections.containExactlyInAnyOrder
```

Comment:
Commit message: Could you provide a teaser in what context such a function could become useful?